extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -414,10 +438,16 @@ export class AmpGeo extends AMP.BaseElement {
             isArray(ampGeoPresets[country]),
             `<amp-geo> preset ${country} not found`
           );
+
           return countries.concat(ampGeoPresets[country]);
         }
+
         // Otherwise we add the country to the list
-        if (country == 'unknown' || /^[a-zA-Z]{2}$/.test(country)) {
+        if (
+          country == VALUE_UNKNOWN ||
+          /^[a-zA-Z]{2}$/.test(country) ||
+          /^[a-zA-Z]{2}-[0-9a-zA-Z]{1,3}$/.test(country)
```

Comment:
following up on @jridgewell 's request here as well to merge the regex

Reply:
Please check